inst/examples/areaChart/app.R

```diff
@@ -112,4 +112,4 @@ server <- function(input, output, session) {
   })
 }
 
-shinyApp(ui, server)
+shinyApp(ui, server, options = list(port = 8888))
```

Comment:
Let's not use the hardcoded default port in all the examples. Can we just specify this using `options(shiny.port = 8765)` or just provide it once in the package.json?

Reply:
Nice idea!